libnavigation-util/src/main/java/com/mapbox/navigation/utils/timer/MapboxTimer.kt

```diff
@@ -13,23 +14,21 @@ import kotlinx.coroutines.launch
  * @param restartAfter Time delay until the timer should restart.
  * @param executeLambda lambda function that is to be executed after [restartAfter] milliseconds.
  */
-class MapboxTimer(private val restartAfter: Long, private val executeLambda: () -> Unit) {
-    private val mainControllerJobScope = ThreadController.getMainScopeAndRootJob()
+class MapboxTimer {
+    private val jobControl = ThreadController.getMainScopeAndRootJob()
 
-    private val timerJob: Job by lazy {
-        mainControllerJobScope.scope.launch {
+    var restartAfter = TimeUnit.MINUTES.toMillis(1)
```

Comment:
Should we rename it to `restartAfterMillis` or smth like it to show info about used time unit in variable name? wdyt?

Reply:
i'm also a fan of putting units on all the variables

Reply:
Yeah that's a good point. Great catch!